be/src/exec/pipeline/scan/morsel.h

```diff
@@ -422,6 +422,11 @@ class BucketSequenceMorselQueue : public MorselQueue {
     Status append_morsels(Morsels&& morsels) override { return _morsel_queue->append_morsels(std::move(morsels)); }
     Type type() const override { return BUCKET_SEQUENCE; }
 
+    void set_tablet_schema(const TabletSchemaCSPtr& tablet_schema) override {
```

Comment:
It seems we also need override
`virtual void set_key_ranges(TabletReaderParams::RangeStartOperation_range_start_op,TabletReaderParams::RangeEndOperation _range_end_op,std::vector<OlapTuple>_range_start_key,std::vector<OlapTuple> _range_end_key) {}`

 introduced in #41348